common/test/acceptance/tests/studio/test_studio_course_info.py

```diff
@@ -0,0 +1,154 @@
+            And I should see the new update button
+        """
+        self.course_updates_page.visit()
+        self.course_updates_page.wait_for_page()
+        self.assertTrue(self.course_updates_page.are_course_updates_on_page())
+        self.assertTrue(self.course_updates_page.is_new_update_button_present)
+
+    def test_new_course_update_is_present(self):
+        """
+          Scenario: Users can add updates
+              Given I have opened a new course in Studio
+              And I go to the course updates page
+              When I add a new update with the text "Hello"
+              Then I should see the update "Hello"
+              And I see a "saving" notification
+        """
+        self.course_updates_page.visit()
+        self.assertTrue(self.course_updates_page.is_new_update_button_present())
+        self.course_updates_page.click_new_update_button()
+        self.course_updates_page.submit_update('Hello')
+        self.assertTrue(self.course_updates_page.update_text_contains('Hello'))
+
+    def test_new_course_update_can_be_edited(self):
+        """
+        Scenario: Users can edit updates
+            Given I have opened a new course in Studio
+            And I go to the course updates page
+            When I add a new update with the text "Hello"
+            And I modify the text to "Goodbye"
+            Then I should see the update "Goodbye"
+        """
+
+        self.course_updates_page.visit()
+        self.assertTrue(self.course_updates_page.is_new_update_button_present())
+        self.course_updates_page.click_new_update_button()
+        self.course_updates_page.submit_update('Hello')
+        self.assertTrue(self.course_updates_page.update_text_contains('Hello'))
+        self.assertTrue(self.course_updates_page.is_edit_button_present())
+        self.course_updates_page.click_edit_update_button()
+        self.course_updates_page.submit_update('Goodbye')
+        self.assertFalse(self.course_updates_page.update_text_contains('Hello'))
+        self.assertTrue(self.course_updates_page.update_text_contains('Goodbye'))
+
+    def test_delete_course_update(self):
+        """
+        Scenario: Users can delete updates
+              Given I have opened a new course in Studio
+              And I go to the course updates page
+              And I add a new update with the text "Hello"
+              And I delete the update
+              And I confirm the prompt
+              Then I should not see the update "Hello"
+        """
+        self.course_updates_page.visit()
+        self.assertTrue(self.course_updates_page.is_new_update_button_present())
+        self.course_updates_page.click_new_update_button()
+        self.course_updates_page.submit_update('Hello')
+        self.assertTrue(self.course_updates_page.update_text_contains('Hello'))
+        self.course_updates_page.click_delete_update_button()
+        self.assertFalse(self.course_updates_page.update_text_contains('Hello'))
```

Comment:
As mentioned above, here you should verify that there are no updates instead of checking the text.